tests/common.py

```diff
@@ -2,7 +2,7 @@
 import subprocess
 import logging
 
-CORES = "2"
+CORES = ""
```

Comment:
are you sure you want to have it empty with no defaults?

Reply:
not quite sure if I want to determine the number of cores or use the max offered by the system (empty string). Another option would be to parallelise via pytest, but then I'll have to ensure that fixture scopes are maintained. For now I will revert back to 2 cores